javascript/packages/orchestrator/src/test-runner/assertions.ts

```diff
@@ -79,6 +79,50 @@ const Report = ({
   };
 };
 
+const CalcMetrics = ({
+  node_name,
+  metric_name,
+  math_ops,
+  metric_name_calc,
+  target_value,
+  op,
+  timeout,
+}: FnArgs) => {
+  const comparatorFn = comparators[op!];
+  return async (network: Network) => {
+    const nodes = network.getNodes(node_name!);
+    const promise1 = nodes.map((node: any) =>
+      node.getMetric(
+        metric_name!,
+        toChaiComparator(op!),
+        target_value!,
+        timeout || DEFAULT_INDIVIDUAL_TEST_TIMEOUT,
+      ),
+    );
+
+    const promise2 = nodes.map((node: any) =>
+      node.getMetric(
+        metric_name_calc!,
+        toChaiComparator(op!),
+        target_value!,
+        timeout || DEFAULT_INDIVIDUAL_TEST_TIMEOUT,
+      ),
+    );
+
+    const values = await Promise.all([...promise1, ...promise2]);
```

Comment:
In theory we can have `groups` here where we get more than two values, that why we get an array of promises.
We should cover that case by looping (by two) the `values` and checking that for every pair the comparison success.